### PR TITLE
tools/export: add hello_zig support in kernel mode for linux host

### PR DESCRIPTION
**Pull request by yf13**
## Summary

Export Zig related macros to `nuttx-export-x.y.z.tar.gz` for apps side to build hello_zig app on Linux host.

## Impact

None

## Testing

checked with `rv-virt/knsh32`

### CODE REVIEW

File: tools/Zig.defs [hunk 1]
```diff
@@ -39,4 +39,6 @@ endif
 
 # Convert cortex-xxx/sifive-exx to cortex_xxx/sifive_exx
 
+ifneq ($(LLVM_CPUTYPE),)
 ZIGFLAGS += -mcpu $(subst -,_,$(LLVM_CPUTYPE))
+endif
```
Comment (acassis) [inline tools/Zig.defs:44]: Any idea why only Zig does this kind of substitution for LLVM?

Comment (yf13) [reply tools/Zig.defs:44]: No idea, it was there before and I got empty string during local test, so I guarded it to move ahead.

Comment (yf13) [reply tools/Zig.defs:44]: @acassis it seems that "sim/wamr" is blocking the CI checks, will push again later until it is resolved.

**Comment by xiaoxiang781216** [comment]
please rebase to the last master, ci break is fixed.@yf13

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**